### PR TITLE
LPS-159414 & LPS-149437 [App Pages] Show alert when deleting required input fragments

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
@sandrodw3 

[a407d56](https://github.com/liferay-echo/liferay-portal/pull/9161/commits/a407d561cba7edec7d7bb2d86365aa82b010dd9e)

🤔

**Review by p2kmgcl — COMMENTED** [review]
Just a tiny typo. Feel free to fix that in another PR :heart:

**Comment by p2kmgcl** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getFormValidationData.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export const FORM_VALIDATION_TYPES = {
+	deletedField: 'deletedField',
+	deletedFragment: 'deletedFragment',
+	hiddenFields: 'hiddenFragments',
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getFormValidationData.js:18]: Oooh I'll fix it in the next PR 🥰

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getFormValidationData.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	hiddenFragment: 'hiddenFragment',
+	missingFields: 'missingFields',
+	missingFragments: 'missingFragments',
+	missingSubmit: 'missingSubmit',
+};
+
+export function getFormValidationData({name = null, type}) {
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getFormValidationData.js:25]: :peeposhy: ❤️

**Comment by sandrodw3** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:reevaluate:1327401_5078

**Comment by lfbesada** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
@sandrodw3 rebase errors :elmo:

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:reevaluate:1267401_4320

**Comment by lfbesada** [comment]
ci:forward

**Pull request closed without merging**